### ISSUE TITLE
I had some problems when compiling with ninja

### ISSUE THREAD

**Issue by Harry710887048** [zhaofuq/Instant-NSR#7]
I had some problems when compiling the ninja.I also tried different versions of cuda and pytorch, but it did not seem to solve the problem.How could I sovle this problem?
![2962f7b97d0dbf955242027a3c10822](https://user-images.githubusercontent.com/75151571/204088374-049a7164-d681-4dd0-b03b-ff35538e9c49.png)
![50f23058edc8f81dc5e6ab8e77b00e1](https://user-images.githubusercontent.com/75151571/204088410-7dad5890-d15e-4273-b186-2127a4d0a540.png)

**Comment by rulke** [comment]
Can you tell me how to solve it? thanks

**Comment by Harry710887048** [comment]
> Can you tell me how to solve it? thanks

This may be a problem with your tcnn or cuda.Please try to reinstall tcnn like this:
https://github.com/NVlabs/tiny-cuda-nn

**Comment by rulke** [comment]
> > Can you tell me how to solve it? thanks
> 
> This may be the problem with your tcnn or cuda.Please try to reinstall tcnn like this: https://github.com/NVlabs/tiny-cuda-nn

Excuse me, did you finally configure the environment successfully? Can you run?

**Comment by Harry710887048** [comment]
> > > Can you tell me how to solve it? thanks
> > 
> > 
> > This may be the problem with your tcnn or cuda.Please try to reinstall tcnn like this: https://github.com/NVlabs/tiny-cuda-nn
> 
> Excuse me, did you finally configure the environment successfully? Can you run?

Yeah,but I failed to run the gui_nerf.py.I still don't know how to training data in dynamic scene.

**Comment by rulke** [comment]
> 



> > > > Can you tell me how to solve it? thanks
> > > 
> > > 
> > > This may be the problem with your tcnn or cuda.Please try to reinstall tcnn like this: https://github.com/NVlabs/tiny-cuda-nn
> > 
> > 
> > Excuse me, did you finally configure the environment successfully? Can you run?
> 
> Yeah,but I failed to run the gui_nerf.py.I still don't know how to training data in dynamic scene.

Please tell me how you solved it? The version is lowered or other things have been modified. What is the composition of your environment? thanks

**Comment by Harry710887048** [comment]
> > 
> 
> > > > > Can you tell me how to solve it? thanks
> > > > 
> > > > 
> > > > This may be the problem with your tcnn or cuda.Please try to reinstall tcnn like this: https://github.com/NVlabs/tiny-cuda-nn
> > > 
> > > 
> > > Excuse me, did you finally configure the environment successfully? Can you run?
> > 
> > 
> > Yeah,but I failed to run the gui_nerf.py.I still don't know how to training data in dynamic scene.
> 
> Please tell me how you solved it? The version is lowered or other things have been modified. What is the composition of your environment? thanks

cuda:11.6
pytorch:1.10
ubuntu20.04
python3.7
